lilach-client/src/main/java/org/cshaifasweng/winter/models/Cart.java

```diff
@@ -0,0 +1,15 @@
+package org.cshaifasweng.winter.models;
+
+import java.util.List;
+
+public class Cart {
```

Comment:
This is not an excuse. You can just check if the list is empty.I don't think we really need a Cart entity, and unless you convince me otherwise I want you to remove it.